jcl/src/java.base/share/classes/java/lang/ref/Reference.java

```diff
@@ -60,7 +60,7 @@ public abstract sealed class Reference<T> extends Object permits PhantomReferenc
 
 	private T referent;
 	private ReferenceQueue queue;
-	private int state;
+	volatile private int state;
```

Comment:
Please list modifiers in canonical order:
```java
	private volatile int state;
```

Reply:
updated